scripts/generate_upgrade_syncset.py

```diff
@@ -0,0 +1,301 @@
+#!/bin/env python
+
+import argparse
+import datetime
+import json
+import logging
+import os
+import re
+import subprocess
+import sys
+import ruamel.yaml as yaml
+from ruamel.yaml.scalarstring import DoubleQuotedScalarString as dq
+
+CS = yaml.comments.CommentedSeq
+CM = yaml.comments.CommentedMap
+
+# Script to generate the cluster list
+CLUSTER_LIST_EXEC = 'get-cluster-list.sh'
+# SyncSet CR constants
+SS_API_VERSION = 'hive.openshift.io/v1'
+SS_NAME = 'osd-upgrade-config'
+# UpgradeConfig CR constants
+UC_NAME = 'osd-upgrade-config'
+UC_NAMESPACE = 'openshift-managed-upgrade-operator'
+UC_API_VERSION = 'upgrade.managed.openshift.io/v1alpha1'
+UC_PDB_DRAIN_TIMEOUT = 60
+UC_FORCE = False
+
+def get_cluster_info(cluster_ids, groups, subgroup, environment):
+    """
+    Returns a list of clusters matching the supplied group and subgroup
+    :param group: Cluster group
+    :param subgroup: Cluster subgroup
+    :return: list of tuple pairs of cluster ID and cluster name
+    """
+
+    cluster_id_to_name = dict()
+
+    if cluster_ids is not None:
+        for cluster_id in cluster_ids:
+            try:
+                cluster_name = get_cluster_name_for_id(cluster_id, environment)
+                cluster_id_to_name[cluster_id] = cluster_name
+            except Exception as err:
+                logging.error('Unable to determine cluster name for cluster ID \'{}\': {}'.format(cluster_id, str(err)))
+                sys.exit(1)
+
+    if groups is not None:
+        for g in groups:
+            try:
+                # pout = subprocess.check_output(['cat', '/home/mbargenq/scratch/' + g + '.txt']).decode('utf-8')
```

Comment:
nuke this :)

Reply:
:facepalm: How embarassing! Fixed in 0acedad